token/js/src/extensions/tokenMetadata/state.ts

```diff
@@ -0,0 +1,105 @@
+import type { Commitment, Connection } from '@solana/web3.js';
+import { PublicKey } from '@solana/web3.js';
+import type { TokenMetadata } from '@solana/spl-token-metadata';
+import { Field, unpack } from '@solana/spl-token-metadata';
+
+import { TOKEN_2022_PROGRAM_ID } from '../../constants.js';
+import { ExtensionType, getExtensionData } from '../extensionType.js';
+import { getMint } from '../../state/mint.js';
+
+export const getNormalizedTokenMetadataField = (field: Field | string): string => {
```

Comment:
would have put this in `@solana/spl-token-metadata` but not sure if its worth the admin overhead to publish a new version

Reply:
Actually I think it's fine to live in the token library. The metadata library already does its own "normalizing" within the `FieldCodec`, so it doesn't need this. Token needs this, and it shouldn't be exported, so it's fine here!